conformance/driver.go

```diff
@@ -130,6 +132,7 @@ func (d *Driver) ExecuteMessage(bs blockstore.Blockstore, preroot cid.Cid, epoch
 		Syscalls:       mkFakedSigSyscalls(vm.Syscalls(ffiwrapper.ProofVerifier)), // TODO always succeeds; need more flexibility.
 		CircSupplyCalc: nil,
 		BaseFee:        BaseFee,
+		NtwkVersion:    sm.GetNtwkVersion,
```

Comment:
Right now, `GetNtwkVersion` is a pure function. If we expect it to actually depend on state, this will get more complicated.

Reply:
Any way we could construct an actual state manager? I guess this is _just_ for testing so it's not too bad.

Reply:
`NewStateManager` requires a `ChainStore`, but the kind of vector that gets executed here (message-class vector) doesn't care about the chain. I guess we could create an empty `ChainStore`, but the result at this stage would be the same.

Reply:
I guess we have `GetNtwkVersion` as part of the `StateManager` to be able to upgrade network versions based on things like on-chain voting/signalling?

Reply:
For those cases, we would regardless need to provide a stub/test-controlled `NtwkVersionGetter`, at least for this kind of vector.

Reply:
> I guess we have GetNtwkVersion as part of the StateManager to be able to upgrade network versions conditionally, based on things like on-chain voting/signalling?

Correct